app/artifact-cas/internal/conf/conf.proto

```diff
@@ -43,10 +43,16 @@ message Server {
     string addr = 2;
     google.protobuf.Duration timeout = 3;
   }
+  message TLS {
+    // path to certificate and private key
+    string Certificate = 1;
```

Comment:
this should be lowercase, I believe that the linter didn't pick it up in the CI because there is a bug and `buf lint` doesn't pick the `config` directory.

Note in local how `make lint` or `buf lint app/controlplane/internal/conf` actually should complain. I'll fix the CI issue.

Reply:
Thx. I've updated the file.